### PR TITLE
Header, Footer: Replace hardcoded name

### PR DESCRIPTION
**Pull request by GustavoGomesRibeiro**
footer, header: replace hardcoded name and fetch from constants

### CODE REVIEW

File: src/constants/index.js [hunk 1]
```diff
@@ -795,3 +795,10 @@ export const clients = [
     logo: dropbox,
   },
 ];
+
+export const aboutMe = [
+  {
+    name: "Parth",
+    lastName: "Mittal"
+  }
+];
```
Comment (mittal-parth) [inline src/constants/index.js:804]:
```suggestion
export const aboutMe = {
    name: "Parth Mittal"
  }
```

I feel an object is more extensible as indices won't be required.

File: src/components/Hero.jsx [hunk 2]
```diff
@@ -3,6 +3,7 @@ import LetsConnect from "./LetsConnect";
 import Lottie from "react-lottie";
 import animationData from "../lotties/person-coding.json";
 import { motion } from "framer-motion";
+import { aboutMe } from "../constants";
```
Comment (mittal-parth) [inline src/components/Hero.jsx:6]: Use one import statement for all variables required from `constants`

File: src/components/Hero.jsx [hunk 3]
```diff
@@ -38,7 +39,7 @@ const Hero = () => {
         </div>
 
         <h1 className="font-poppins font-semibold ss:text-[68px] text-[52px] text-white ss:leading-[80px] leading-[80px] w-full">
-          <span className="text-gradient">Parth Mittal </span>
+          <span className="text-gradient">{aboutMe[0].name} {aboutMe[0].lastName}</span>
```
Comment (mittal-parth) [inline src/components/Hero.jsx:42]: Change according to the object notation.

File: src/components/Footer.jsx [hunk 4]
```diff
@@ -10,7 +11,7 @@ const Footer = () => (
     <div className={layout.sectionReverse}>
       <div className={`px-6 ${layout.sectionInfo} md:ml-16`}>
         <h2 className="text-xl font-bold text-gray-800 font-poppins dark:text-white hover:text-gray-700 dark:hover:text-gray-300">
-          Parth Mittal
+          {aboutMe[0].name} {aboutMe[0].lastName}
```
Comment (mittal-parth) [inline src/components/Footer.jsx:14]: Change here as well according to `aboutMe` object.

**Review by mittal-parth — CHANGES_REQUESTED** [review]
@GustavoGomesRibeiro left some review comments. 
Also, rebase your branch off main.

File: src/components/Hero.jsx (hunk 2 above)
Comment (GustavoGomesRibeiro) [reply src/components/Hero.jsx:6]: Here we use just one import to variables from `constants`.
I changed the import statement in the file `Footer.jsx`

**Review by GustavoGomesRibeiro — COMMENTED** [review]
Adjustments in some codes according to review

File: src/components/Footer.jsx (hunk 4 above)
Comment (GustavoGomesRibeiro) [reply src/components/Footer.jsx]: done!

File: src/components/Hero.jsx (hunk 3 above)
Comment (GustavoGomesRibeiro) [reply src/components/Hero.jsx]: done!

File: src/constants/index.js (hunk 1 above)
Comment (GustavoGomesRibeiro) [reply src/constants/index.js]: done!

File: src/components/Hero.jsx [hunk 5]
```diff
@@ -2,6 +2,9 @@ import styles from "../style";
 import LetsConnect from "./LetsConnect";
 import Lottie from "react-lottie";
 import animationData from "../lotties/person-coding.json";
+import { motion } from "framer-motion";
```
Comment (mittal-parth) [inline src/components/Hero.jsx:5]: unused import, not required

Comment (mittal-parth) [inline src/components/Hero.jsx:5]:
```suggestion
```

**Comment by mittal-parth** [comment]
@GustavoGomesRibeiro awesome! Please edit the name and description of the PR as well.

**Review by GustavoGomesRibeiro — COMMENTED** [review]
Done.

I do the rebase and change title and description.

**Review by mittal-parth — APPROVED** [review]

**Comment by mittal-parth** [comment]
@GustavoGomesRibeiro great, merging!
Thanks for contributing! ✨

**Pull request merged**